server/memphis_helper.go

```diff
@@ -426,6 +429,43 @@ func tryCreateInternalJetStreamResources(s *Server, retentionDur time.Duration,
 		DLS_UNACKED_CONSUMER_CREATED = true
 	}
 
+	// create schemaverse dls stream
+	if !DLS_SCHEMAVERSE_STREAM_CREATED {
```

Comment:
You are missing the place that identifies if the stream/consumer was already created beforehand. Do a search DLS_UNACKED_CONSUMER_CREATED = true for example and you will find it in jetstream.go file